src/SFA.DAS.ApplyService.InternalApi.UnitTests/GatewayApiChecksServiceTests.cs

```diff
@@ -22,7 +22,7 @@ namespace SFA.DAS.ApplyService.InternalApi.UnitTests
     public class GatewayApiChecksServiceTests
     {
         private Mock<CompaniesHouseApiClient> _companiesHouseApiClient;
-        private Mock<CharityCommissionApiClient> _charityCommissionApiClient;
+        private Mock<OuterApiClient> _charityCommissionApiClient;
```

Comment:
You perhaps want to rename the variable as well.

Reply:
Done